src/react-app-env.d.ts

```diff
@@ -70,6 +70,7 @@ declare module "math-random-seed";
 declare var MathJax: any;
 declare var ISAAC_SITE: string;
 declare var REACT_APP_API_VERSION: string;
+declare var GOOGLE_RECAPTCHA_SITE_KEY: string;
```

Comment:
I know you're just replicating the style of the other values nearby, but shouldn't this be either `let` or `const`, rather than `var`?

Reply:
Yeah most of those will get cleaned up when linting is tackled, but I've changed them over to const here